src/main/java/com/sysc4806/PostController.java

```diff
@@ -61,8 +61,11 @@ public String postNewAma(@RequestParam(value="title") String title,
 
         Post p = new Post(title, AuthenticationController.CurrentUser(), description);
         ArrayList<String> tagsTrimmed = new ArrayList<>();
-        for(String s : tags.split(","))
-            tagsTrimmed.add(s.trim());
+        //Check if tags is empty or all spaces
+        if(!tags.isEmpty() && !tags.matches("( )*")) {
```

Comment:
Would this match other types of whitespace? I think \s can detect all whitespace, not just spaces